.circleci/config.yml

```diff
@@ -3,14 +3,13 @@
 # Check https://circleci.com/docs/2.0/language-go/ for more details
 version: 2
 jobs:
-  build:
+  test:
     docker:
-      # specify the version
```

Comment:
Gratuitous change, here we specify the Golang version and I have add the comment for a reason.